### DISCUSSION TITLE
Intention behind the "?" and Coffee cards?

### DISCUSSION THREAD

**Discussion by byamagata** [microsoft/azure-boards-estimate#53] [Q&A] [answered]
I have no idea what the intention was behind these cards, would love to know what others/the devs were thinking when adding them.

**Comment by FreakinWard** [comment, marked as answer]
> I have no idea what the intention was behind these cards, would love to know what others/the devs were thinking when adding them.

Solid question, @byamagata. I've used them in the past but are rare. 

It would be neat if the definitions were more easily accessible. Maybe a hover of sorts.

While I'm uncertain as to the teams definition, [wikipedia](https://en.wikipedia.org/wiki/Planning_poker#:~:text=optionally%20a%C2%A0%3F%20(unsure)%2C%20an%20infinity%20symbol%20(this%20task%20cannot%20be%20completed)%2C%20and%20a%20coffee%20cup%20(I%20need%20a%20break%2C%20and%20I%20will%20make%20the%20rest%20of%20the%20team%20coffee)) definition aligns with the way I've understood them.

- The question mark is used when an estimator has not enough information to make an estimate. 
- The coffee break card is used when an estimator needs a break.

**Comment by ahmadfaiqateel** [comment]
What does the pretzel mean?
